### ISSUE TITLE
[Bug Report] ttnn.sigmoid returns all zeros for certain input

### ISSUE THREAD

**Issue by bfilipovicTT** [tenstorrent/tt-metal#19976]
Describe the bug
ttnn.sigmoid return asll zeroes for a certain input found in UNet-brain model

Input spec:

    inputs can be read from csv file
    input shape: [1,1,256,256]
    datatype: ttnn.bfloat16
    layout: ttnn.TILE_LAYOUT

**To Reproduce**
1. download the csv file with inputs and run the unit test:

```
from loguru import logger
import pytest
import torch
import ttnn
import traceback
from tests.ttnn.utils_for_testing import assert_with_pcc
aten = torch.ops.aten


def max_relative_error(expected, calculated):
    assert expected.shape == calculated.shape, "Tensors must have the same shape"
    if expected.dtype == torch.bool:
        expected = expected.to(torch.int)
    if calculated.dtype == torch.bool:
        calculated = calculated.to(torch.int)
    relative_error = torch.abs(expected - calculated) / (torch.abs(expected) + 1e-8)
    mre = torch.max(relative_error).item()
    inds = torch.where(relative_error == mre)
    return mre, inds

def load_tensor_from_csv(filename, shape):
    import pandas as pd
    df = pd.read_csv(filename)
    data = df.values  
    tensor = torch.tensor(data, dtype=torch.bfloat16)  
    tensor = tensor.view(*shape)  
    return tensor

def run_sigmoid_tests(
    input_shape, 
    dtype, 
    dlayout,
    filename,
    device,
):

    x = load_tensor_from_csv(filename, input_shape)

    try:
        ref_value = aten.sigmoid.default(x)
        
        tt_x = ttnn.from_torch(x, dtype=dtype[0], layout=dlayout[0], device=device)

        tt_result = ttnn.sigmoid(tt_x)
        tt_result = ttnn.to_torch(tt_result)
    except Exception as e:
        logger.warning(f"Test execution crashed: {e}")
        print(traceback.format_exc())
        raise e

    assert len(tt_result.shape) == len(ref_value.shape)
    assert tt_result.shape == ref_value.shape
    assert_with_pcc(ref_value, tt_result, 0.999)


test_sweep_args = [
    (
        [[1,1,256,256]],
        [ttnn.bfloat16],
        [ttnn.TILE_LAYOUT],
        "UNET_BRAIN_sigmoid0_input.csv",
    ),
]


@pytest.mark.parametrize(
    "input_shape, dtype, dlayout, filename",
    (test_sweep_args),
)
def test_sigmoid(input_shape, dtype, dlayout, filename):
    device = ttnn.open_device(device_id=0)
    run_sigmoid_tests(input_shape, dtype, dlayout, filename, device)
    ttnn.close_device(device)
```

CSV with inputs:
[UNET_BRAIN_sigmoid0_input.csv](https://github.com/user-attachments/files/19551956/UNET_BRAIN_sigmoid0_input.csv)

**Comment by mouliraj-mcw** [comment]
Hi @bfilipovicTT,
It looks like the issue is occurring only for negative values. Could you try using sigmoid_accurate instead of sigmoid and check if that improves the results?

**Comment by bfilipovicTT** [comment]
Yes, sigmoid_accurate solves the sigmoid issue

**Comment by mouliraj-mcw** [comment]
Hi @bfilipovicTT.
Can we close this issue.
